src/command_modules/azure-cli-sql/HISTORY.rst

```diff
@@ -2,9 +2,11 @@
 
 Release History
 ===============
+
 2.2.1
 +++++
 * Updating az sql mi create and update with two new properties (proxyOverride and publicDataEndpointEnabled)
+* Allow elastic pool name (instead of id) to be specified for az sql db update.
```

Comment:
Just to be certain, the corrected behavior will allow elastic pool name or ID, correct?

Reply:
That's right

Reply:
Let me double check that there is test coverage of both

Reply:
Updated test and tried to clarify history entry